project/AppDependencies.scala

```diff
@@ -3,12 +3,12 @@ import sbt.*
 object AppDependencies {
 
   private lazy val mongoHmrcVersion     = "2.2.0"
-  private lazy val bootstrapPlayVersion = "9.2.0"
+  private lazy val bootstrapPlayVersion = "9.3.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.6.0"
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.8.0"
```

Comment:
Please update to `10.9.0` when making the correction

Reply:
Done